Themes/Flatly/views/layouts/master.blade.php

```diff
@@ -7,9 +7,11 @@
     @show
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>@section('title')@setting('core::site-name')@show</title>
-    @foreach($alternate as $alternateLocale=>$alternateSlug)
-        <link rel="alternate" hreflang="{{$alternateLocale}}" href="{{url($alternateLocale.'/'.$alternateSlug)}}">
-    @endforeach
+    @if(isset($alternate))
+    	@foreach($alternate as $alternateLocale=>$alternateSlug)
+        	<link rel="alternate" hreflang="{{$alternateLocale}}" href="{{url($alternateLocale.'/'.$alternateSlug)}}">
```

Comment:
Seems like there's a mix of tabs and spaces. Please use 4 spaces instead. Same on line 11.